src/flask_pyoidc/flask_pyoidc.py

```diff
@@ -102,14 +217,21 @@ def _handle_authentication_response(self):
         # do token request
         args = {
             'code': authn_resp['code'],
-            'redirect_uri': self.client.registration_response['redirect_uris'][0],
+            'redirect_uri':
+                self.client.registration_response['redirect_uris'][0]
```

Comment:
nit: should this be passed to the function instead? ie allow to pick which redirect uri instead of the first one, if needed by users of the library